test/cql-pytest/test_describe.py

```diff
@@ -0,0 +1,344 @@
+                dclocal_read_repair_chance, default_time_to_live, extensions,
+                flags, gc_grace_seconds, max_index_interval,
+                memtable_flush_period_in_ms, min_index_interval, read_repair_chance,
+                speculative_retry
+                FROM system_schema.tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(tbl)}'
+            """).one()
+            new_tbl_row = cql.execute(f"""
+                SELECT keyspace_name, bloom_filter_fp_chance, caching,
+                comment, compaction, compression, crc_check_chance,
+                dclocal_read_repair_chance, default_time_to_live, extensions,
+                flags, gc_grace_seconds, max_index_interval,
+                memtable_flush_period_in_ms, min_index_interval, read_repair_chance,
+                speculative_retry
+                FROM system_schema.tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(new_tbl)}'
+            """).one()
+            assert tbl_row == new_tbl_row
+
+            tbl_scylla_row = cql.execute(f"""
+                SELECT cdc, partitioner
+                FROM system_schema.scylla_tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(tbl)}'
+            """).one()
+            new_tbl_scylla_row = cql.execute(f"""
+                SELECT cdc, partitioner
+                FROM system_schema.scylla_tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(new_tbl)}'
+            """).one()
+            assert tbl_scylla_row == new_tbl_scylla_row
+        finally:
+            cql.execute(f"DROP TABLE {new_tbl}")
+
+# Test that `DESC TYPE {udt}` contains appropriate create statement for user-defined type
+def test_desc_type(cql, test_keyspace, randomizer_seed):
+    with new_random_type(cql, test_keyspace) as udt:
+        desc = cql.execute(f"DESC TYPE {udt}")
+        desc_stmt = desc.one().create_statement
+
+        new_udt = f"{test_keyspace}.{unique_name()}"
+        new_desc_stmt = desc_stmt.replace(udt, new_udt)
+
+        try:
+            cql.execute(new_desc_stmt)
+            udt_row = cql.execute(f"""
+                SELECT keyspace_name, field_names, field_types
+                FROM system_schema.types
+                WHERE keyspace_name='{test_keyspace}' AND type_name='{get_name(udt)}'
+            """).one()
+            new_udt_row = cql.execute(f"""
+                SELECT keyspace_name, field_names, field_types
+                FROM system_schema.types
+                WHERE keyspace_name='{test_keyspace}' AND type_name='{get_name(new_udt)}'
+            """).one()
+            assert udt_row == new_udt_row
+        finally:
+            cql.execute(f"DROP TYPE {new_udt}")
+
+# Test that `DESC FUNCTION {udf}` contains appropriate create statement for user-defined function
+def test_desc_function(cql, test_keyspace, randomizer_seed):
```

Comment:
This test cannot be run on Cassandra (i.e., wants "scylla_only") because Cassandra doesn't support Lua.
Please check that the other tests pass on Cassandra, and if not - why not. (run it with test/cql-pytest/run-cassandra).

Reply:
Added. Also this fixture was required in tests like `test_desc_table` and `test_desc_keyspace` because we have scylla-specific informations (eg. in table `system_schema.scylla_tables`).

For now I've also added this fixture to `test_tables` test, which validates `DESC TABLES` statement. 
This is because C* for some reason quotes `IndexInfo` table (from `system` keyspace). I don't see why this is needed and my implementation doesn't quote this, so this test doesn't pass on C*

Reply:
@Jadw1 if there's a test that after careful consideration you decide that it fails on Cassandra because it's a Cassandra bug - and the behavior is correct on Scylla, please add to it the"cassandra_bug" fixture and explain this in a comment. We have several cases of cassandra_bug in the test suite already, when our testing discovered what we consider bugs - not features - in Cassandra. But please consider carefully each case whether this is really a bug in Cassandra and not correct behavior that for some reason (?) we decided to change.